skills/csharp/calendarskill/Dialogs/MainDialog.cs

```diff
@@ -99,16 +99,20 @@ protected override async Task<DialogTurnResult> OnBeginDialogAsync(DialogContext
                 // Get cognitive models for the current locale.
                 var localizedServices = _services.GetCognitiveModels();
 
-                // Run LUIS recognition on Skill model and store result in turn state.
-                localizedServices.LuisServices.TryGetValue("Calendar", out var skillLuisService);
-                if (skillLuisService != null)
+                var luisResult = innerDc.Context.TurnState.Get<CalendarLuis>(StateProperties.CalendarLuisResultKey);
```

Comment:
When will CalendarLuis be generated before OnBeginDialogAsync? Thanks

Reply:
In Building/FloorNumer prompt